src/test/java/com/kainos/ea/integration/JobRolesIntegrationTest.java

```diff
@@ -0,0 +1,45 @@
+package com.kainos.ea.integration;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.kainos.ea.WebServiceApplication;
+import com.kainos.ea.WebServiceConfiguration;
+import com.kainos.ea.model.JobRole;
+import io.dropwizard.configuration.ResourceConfigurationSourceProvider;
+import io.dropwizard.testing.junit5.DropwizardAppExtension;
+import io.dropwizard.testing.junit5.DropwizardExtensionsSupport;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import javax.ws.rs.core.Response;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+@ExtendWith(DropwizardExtensionsSupport.class)
+public class JobRolesIntegrationTest {
+
+    static final DropwizardAppExtension<WebServiceConfiguration> APP = new DropwizardAppExtension<>(
+            WebServiceApplication.class, null,
+            new ResourceConfigurationSourceProvider()
+    );
+
+    @Test
+    void getEmployees_shouldReturnListOfEmployees()  {
+        ObjectMapper mapper = new ObjectMapper();
+        List<JobRole> response = APP.client().target("http://localhost:8080/api/job-roles")
+                .request()
+                .get(List.class);
```

Comment:
If we remove the `.get(List.class)` from here we can have access to the Response to get the status, and create a second variable for the List

```suggestion
        Response response = APP.client().target("http://localhost:8080/api/job-roles")
                .request();
                
      List<JobRole> jobRoles = response.get(List.class)
                
```